README.md

```diff
@@ -16,6 +16,11 @@ This is an Ember CLI addon, to install:
 
 `ember install ember-href-to`
 
+## Supported Ember Versions
+
+- v5.0.1. supports Ember versions >=3.1 & <3.27
```

Comment:
This kind of thing?

"It will work in >3.27 provided you are not using the modernized LinkTo, but are subclassing from the @ember/legacy-built-in-components addon"

Reply:
I guess it might be okay to omit because you'd need to specifically subclass all links using this addon? It doesn't automatically switch them to the non-Glimmer version when the addon is installed in the app?

Reply:
>  It doesn't automatically switch them to the non-Glimmer version when the addon is installed in the app?

No. 

> I guess it might be okay to omit because you'd need to specifically subclass all links using this addon?

Correct, you'd have to subclass them. 

The only edge case scenario is if you are on >=3.27 and <4 and you reopen the built-in LinkComponent (not the one from the addon), it will automatically switch all LinkTo to the legacy implementation. I don't think this is something we need to highlight here though.